packages/shortcuts-extension/test/components/ShortcutInput.spec.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+import React from 'react';
+import { ShortcutInput } from '../../src/components/ShortcutInput';
+import { IShortcutTarget } from '../../src/types';
+import { nullTranslator } from '@jupyterlab/translation';
+import '@testing-library/jest-dom';
```

Comment:
Is this needed? I think we would want to start using `testing-library` as discussed in https://github.com/jupyterlab/jupyterlab/issues/15807 but it seems to me that this test file does not take advantage of it as I do not see custom matchers this particular package provides? I might be wrong.

```suggestion
```

Reply:
@kjayan this is still failing:

```
FAIL  test/components/ShortcutInput.spec.ts
  ● Test suite failed to run

    Cannot find module '@testing-library/jest-dom' from 'test/components/ShortcutInput.spec.ts'

       8 | import { IShortcutTarget } from '../../src/types';
       9 | import { nullTranslator } from '@jupyterlab/translation';
    > 10 | import '@testing-library/jest-dom';
         | ^
      11 | import { PromiseDelegate } from '@lumino/coreutils';
      12 | import { createRoot } from 'react-dom/client';
      13 |

      at Resolver._throwModNotFoundError (../../node_modules/jest-resolve/build/resolver.js:427:11)
      at Object.<anonymous> (test/components/ShortcutInput.spec.ts:10:1)
```